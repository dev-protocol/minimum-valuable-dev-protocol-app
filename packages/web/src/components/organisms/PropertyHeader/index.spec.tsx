import React from 'react'
import { render } from '@testing-library/react'
import { PropertyHeader } from '.'

jest.mock('src/fixtures/dev-kit/hooks')

describe(`${PropertyHeader.name}`, () => {
  test('Snapshot', () => {
    const component = render(<PropertyHeader propertyAddress="propertyAddress" />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
