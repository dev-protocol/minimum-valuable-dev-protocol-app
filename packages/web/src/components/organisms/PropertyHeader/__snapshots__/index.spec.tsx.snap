// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PropertyHeader Snapshot 1`] = `
<body>
  <div>
    <div
      class="sc-AxjAm czRilP"
    >
      <span
        class="heading"
      >
        Property Address
      </span>
      <div
        class="address"
      >
        propertyAddress
      </div>
    </div>
  </div>
</body>
`;
